mycroft/res/text/fa-ir/configurations.json

```diff
@@ -0,0 +1,33 @@
+{
+  "unit system": {
+    "metric": {"system_unit": "metric"},
+    "imperial": {"system_unit": "imperial"}
+  },
+  "location": {
+    "stockholm": {
```

Comment:
this whole file doesn't look right

Reply:
What it should be? I just copy pasted it from `en-us` and other languages seems doesn't have it.

Reply:
That file offers defaults related to the Voight-Kampff test system. Farsi doesn't need a copy.

Reply:
Yeah, it doesn't hurt to have it, but it also won't be used. It's been removed now.

Merging.